jdaviz/core/helpers.py

```diff
@@ -55,14 +56,18 @@ def __init__(self, app=None, verbosity='warning', history_verbosity='info'):
         # give a reference from the app back to this config helper.  These can be accessed from a
         # viewer via viewer.jdaviz_app and viewer.jdaviz_helper
         # if the helper has already been set, this is probably a nested viz tool. Don't overwrite
-        if not hasattr(self.app, '_jdaviz_helper'):
+        if self.app._jdaviz_helper is None:
             self.app._jdaviz_helper = self
 
         self.app.hub.subscribe(self, SubsetCreateMessage,
                                handler=lambda msg: self._propagate_callback_to_viewers('_on_subset_create', msg)) # noqa
         self.app.hub.subscribe(self, SubsetDeleteMessage,
                                handler=lambda msg: self._propagate_callback_to_viewers('_on_subset_delete', msg)) # noqa
 
+        self._in_batch_load = 0
```

Comment:
Not very important, but should we use bools instead of ints here for clarity? The in-place arithmetic that happens [below](https://github.com/spacetelescope/jdaviz/pull/1742/files#diff-cb5e0cef9bcbcb30559a6ce19d4bf05e855cfb36701fb325e690e53aeb0906eaR92) had me scratching my head a bit.

Reply:
the integer allows this to work for nested instances (so that the final logic does not occur until the outermost with statement exits).  Probably not a common use-case, but doesn't hurt?

Reply:
That makes sense! Neat.